### PR TITLE
Setup for Italian translation and add quicktour.mdx translation

### LINKED ISSUE #17459: Tranformers documentation translation to Italian
Hi!

Let's bring the documentation to all the Italian-speaking community :)

Who would want to translate? Please follow the 🤗 [TRANSLATING guide](https://github.com/huggingface/transformers/blob/main/docs/TRANSLATING.md). Here is a list of the files ready for translation. Let us know here if you'd like to translate any and we'll add your name to the list.

Some notes:

- Please translate using an informal tone (imagine you are talking with a friend about transformers 🤗). For example, use Tu instead of Lei.
- Please translate in a gender-neutral way.
- Add your translations to a new folder (create it if it's not already available) called _it_ inside the [source folder](https://github.com/huggingface/transformers/tree/main/docs/source).
- Register your translation in [it/_toctree.yml](https://github.com/huggingface/transformers/blob/main/docs/source/it/_toctree.yml); please follow the order of the [English version](https://github.com/huggingface/transformers/blob/main/docs/source/en/_toctree.yml).
- Once you're finished, open a pull request and tag this issue by including #issue-number in the description, where issue-number is the number of this issue.
- 🙋 If you'd like others to help you with the translation, you can also post in the  🤗 [forums](https://discuss.huggingface.co/).

## Get Started section
- [x] [index.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/index.mdx) WIP @mfumanelli
- [x] [quicktour.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/quicktour.mdx). WIP @mfumanelli
- [ ] [installation.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/installation.mdx). WIP @mfumanelli
## Tutorial section
- [ ] [pipeline_tutorial.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/pipeline_tutorial.mdx) WIP @nickprock
- [ ] [autoclass_tutorial.mdx](https://github.com/huggingface/transformers/blob/master/docs/source/autoclass_tutorial.mdx)
- [ ] [preprocessing.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/preprocessing.mdx)
- [ ] [training.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/training.mdx)
- [ ] [accelerate.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/accelerate.mdx)
- [ ] [model_sharing.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_sharing.mdx)
- [ ] [multilingual.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/multilingual.mdx)

### PR DESCRIPTION
**Pull request by mfumanelli**
I created the folder for files translated into Italian and made the translation of quicktour.mdx and index.mdx files.

The folder (called 'it') is located in transformers/docs/source/.
The translated documents are located in the transformers/docs/source/it folder.

Fixes [#17459](https://github.com/huggingface/transformers/issues/17459)

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?

@omarespejel @sgugger

### CODE REVIEW

**Comment by omarespejel** [comment]
This looks great! Thanks @mfumanelli for the PR and opening the venue for an Italian 🇮🇹 translation!

Since this is the first IT translation could please add `it` to the languages section in the `.github/workflows` ([here](https://github.com/huggingface/transformers/blob/main/.github/workflows/build_documentation.yml) and [here](https://github.com/huggingface/transformers/blob/main/.github/workflows/build_pr_documentation.yml))?

**Review by omarespejel — APPROVED** [review]

**Comment by omarespejel** [comment]
Thank you @mfumanelli 🇮🇹! LGTM, @sgugger :)

File: docs/source/it/_config.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# docstyle-ignore
+INSTALL_CONTENT = """
+# Transformers installation
+! pip install transformers datasets
+# To install from source instead of the last release, comment the command above and uncomment the following one.
```
Comment (sgugger) [inline docs/source/it/_config.py:5]: Those two comments could be translated in Italian as well :-)

**Review by sgugger — APPROVED** [review]
Thanks for adding support for this new language!

File: docs/source/it/_config.py (hunk 1 above)
Comment (mfumanelli) [reply docs/source/it/_config.py]: Done, I hadn't noticed them, thank you 😊 🌈

**Pull request merged**